Dockerfile

```diff
@@ -57,24 +59,31 @@ COPY deploy/ssl /etc/ssl/qpc
 # Create /deploy
 COPY deploy  /deploy
 
+# Copy server code
+COPY . .
+
+# Install quipucords as package
+RUN pip install -v -e .
```

Comment:
I'd rather have this layer as "external" / late as possible - to optimize build time for developers (me, hehe) rebuilding the image frequently.

I THINK this can happen right after `USER discovery`. If not, before `RUN chown ...`